db/migrate/20180422030623_create_slack_user.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSlackUser < ActiveRecord::Migration[5.0]
+  def change
+    create_table :slack_users do |t|
+      t.string :slack_id, index: true
```

Comment:
You'll need to add a column for the `user` association.  i.e.

```ruby
t.integer :user_id
```

Reply:
Added this column. I included index. Is that what you meant?